test/e2e/network_test.go

```diff
@@ -466,8 +466,59 @@ var _ = Describe("Podman network", func() {
 		Expect(lines[1]).To(Equal(netName2))
 	})
 
-	It("podman network with multiple aliases", func() {
-		SkipUntilAardvark(podmanTest)
+	It("podman CNI network with multiple aliases", func() {
+		SkipIfNetavark(podmanTest)
+		var worked bool
+		netName := "aliasTest" + stringid.GenerateNonCryptoID()
+		session := podmanTest.Podman([]string{"network", "create", netName})
+		session.WaitWithDefaultTimeout()
+		defer podmanTest.removeNetwork(netName)
+		Expect(session).Should(Exit(0))
+
+		interval := time.Duration(250 * time.Millisecond)
+		for i := 0; i < 6; i++ {
+			n := podmanTest.Podman([]string{"network", "exists", netName})
+			n.WaitWithDefaultTimeout()
+			worked = n.ExitCode() == 0
+			if worked {
+				break
+			}
+			time.Sleep(interval)
+			interval *= 2
+		}
+
+		top := podmanTest.Podman([]string{"run", "-dt", "--name=web", "--network=" + netName, "--network-alias=web1", "--network-alias=web2", nginx})
+		top.WaitWithDefaultTimeout()
+		Expect(top).Should(Exit(0))
+		interval = time.Duration(250 * time.Millisecond)
+		// Wait for the nginx service to be running
+		for i := 0; i < 6; i++ {
+			// Test curl against the container's name
+			c1 := podmanTest.Podman([]string{"run", "--dns-search", "dns.podman", "--network=" + netName, nginx, "curl", "web"})
+			c1.WaitWithDefaultTimeout()
+			worked = c1.ExitCode() == 0
+			if worked {
+				break
+			}
+			time.Sleep(interval)
+			interval *= 2
+		}
+		Expect(worked).To(BeTrue())
+
+		// Nginx is now running so no need to do a loop
+		// Test against the first alias
+		c2 := podmanTest.Podman([]string{"run", "--dns-search", "dns.podman", "--network=" + netName, nginx, "curl", "web1"})
+		c2.WaitWithDefaultTimeout()
+		Expect(c2).Should(Exit(0))
+
+		// Test against the second alias
+		c3 := podmanTest.Podman([]string{"run", "--dns-search", "dns.podman", "--network=" + netName, nginx, "curl", "web2"})
+		c3.WaitWithDefaultTimeout()
+		Expect(c3).Should(Exit(0))
+	})
+
+	FIt("podman Netavark network with multiple aliases", func() {
```

Comment:
@baude see this test. It first fails at: `top := podmanTest.Podman([]string{"run", "-dt", "--name=web", "--network=" + netName, "--network-alias=web1", "--network-alias=web2", nginx})`.